teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md

```diff
@@ -280,6 +289,20 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Notify
+Sends an email to Teams phone user about new telephone number assignment.
+
+```yaml
+Type: Switch
+Parameter Sets: (Notify)

```

Comment:
Parameter Sets is `Assignment`, it can only be used in the assignment flow `Assignment`.